src/Engine/Renderer/RenderTechnique/ShaderConfiguration.cpp

```diff
@@ -146,28 +145,25 @@ static const std::string defaultFragmentShader(
 
 #else
 /// todo : find a way to define (if this make sense) default shaders.
-static const std::string defaultVertexShader{"Shaders/Default.vert.glsl"};
-static const std::string defaultFragmentShader{"Shaders/Default.frag.glsl"};
+  static const std::string defaultVertexShader{"Shaders/Default.vert.glsl"};
+  static const std::string defaultFragmentShader{"Shaders/Default.frag.glsl"};
 #endif
 
-namespace Ra {
-namespace Engine {
-
 ShaderConfiguration ShaderConfiguration::m_defaultShaderConfig( "Default Program",
                                                                 defaultVertexShader,
                                                                 defaultFragmentShader );
 
-ShaderConfiguration::ShaderConfiguration( const std::string& name ) :
-    m_name{name},
+ShaderConfiguration::ShaderConfiguration(std::string name ) :
+    m_name{std::move(name)},
```

Comment:
Ok, now I am sure you didn't use the code style hook on your code...
Please fix that.
```suggestion
    m_name{std::move( name )},
```